index.html

```diff
@@ -3,10 +3,68 @@
       <head>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
         <link rel="stylesheet" href="styles/main.css">
-        <title>JT</title>
+        <title>JT's Portfolio</title>
       </head>
 <body>
-    <header> <h1>Jud&aacute; Teixeira</h1> </header>
+  
+  <header>
+    <div>
+      <a href="#" class="logo">
+        <img src="images/banner.png" alt="Jt-logo" height="35">
+      </a>
+    </div>
+    
+
+    <div>
+      <nav>
+        <a href="#" class = "menu">
+        <img src="images/menu.png" alt="Menu" height="35">
+        </a>
+     </nav>
+    </div>
+  </header>
+  
+  <div class="bg">
+    <img src="images/bg.png" alt="BG">
+  </div>
+
+  <div class="main">
+  
+  
+  <div class="content">
+
+    <h1>I'm Jud&aacute; <br> Glad to see <br> you!</h1>
+    <p>
+      I'm a brazillian web developer! 
+      I can help you build a product, feature 
+      or website Look through some of my work and experience! 
+      If you like what you see and have a project you need coded, 
+      don't hesitate to contact me.
+      </p>
+
+  </div>
+
+  <div class="footer">
+    <h3>LET'S CONNECT</h3>
+  <div class="icon-bar">
+    <a href="https://twitter.com/87juda">
```

Comment:
- Kindly nest your `<a>` tags in a `<li>` using `<ul>`.
- Provide your design parameters in your `css` file instead of use inline parameters (`height="30"`)

Reply:
Done, thank you!